pyproject.toml

```diff
@@ -70,3 +71,15 @@ quote-style = "double"
 indent-style = "space"
 skip-magic-trailing-comma = false
 line-ending = "auto"
+
+
+[tool.setuptools_scm]
+
+
+[tool.setuptools.packages.find]
+include = ["bayes3d"]
```

Comment:
does this find all subpackages as well? I guess it does thanks to your `__all__` changes?

Reply:
Yup, this setting tells setuptools to include everything under [`bayes3d/bayes3d`](https://github.com/probcomp/bayes3d/tree/main/bayes3d) as part of the package.

Even without having [`__all__` in `bayes3d/__init__.py`](https://github.com/probcomp/bayes3d/blob/597f1f0f2a42cff0ef8f680ebbde8bf8430711d5/bayes3d/__init__.py#L15), it will still discover all subpackages, though :). The reason I added `__all__` there is to hint ruff that we are intentionally importing some of the modules to make them available directly, so that ruff won't complain that they are imported but not used [on this line](https://github.com/probcomp/bayes3d/blob/597f1f0f2a42cff0ef8f680ebbde8bf8430711d5/bayes3d/__init__.py#L5).

Without [`from . import colmap, distributions, scene_graph, utils`](https://github.com/probcomp/bayes3d/blob/597f1f0f2a42cff0ef8f680ebbde8bf8430711d5/bayes3d/__init__.py#L5) in `bayes3d/__init__.py`, the following lines will fail:
```python
import bayes3d as b
b.distributions
```
Output:
```
AttributeError: module 'bayes3d' has no attribute 'distributions'
```
However, `distributions` still exists as part of Bayes3D package. We just need to import it explicitly:
```python
import bayes3d as b
import bayes3d.distributions
b.distributions  # <- this works just fine
```
Output:
```
<module 'bayes3d.distributions' from '/blahblahblah/site-packages/bayes3d/distributions.py'>
```

Since we used to have [`from . import distributions`, etc.](https://github.com/probcomp/bayes3d/pull/42/files#diff-4e6fc4c661b288b53e857ec36751555a40ce477173c7ccd1ff9373f4248a7d0bL10-L13) prior to https://github.com/probcomp/bayes3d/pull/42, and people have been relying on this ([there are many cases where people use `b.distributions` without explicitly importing the submodule](https://github.com/search?q=repo%3Aprobcomp%2Fbayes3d%20b.distributions&type=code)), I just thought it might be a good idea to add those imports back to `bayes3d/__init__.py`.

### TL;DR

1. Yes, this will find all sub-packages, but it's not because of the `__all__` change.
2. The `__all__` change is to make linter happier about `from . import colmap, distributions, scene_graph, utils`
3. The purpose of `from . import colmap, distributions, scene_graph, utils` is to get rid of the need for explicit import and for backward compatibility

Hope this clarifies things a bit @sritchie, and thanks for the review! :D